DigitalOffice/Services/UserService/Repositories/UserRepository.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Linq;
+using UserService.Database;
+using UserService.Database.Entities;
+using UserService.Repositories.Interfaces;
+
+namespace UserService.Repositories
+{
+    public class UserRepository : IUserRepository
+    {
+        private UserServiceDbContext dbContext;
+
+        public UserRepository([FromServices] UserServiceDbContext dbContext)
+        {
+            this.dbContext = dbContext;
+        }
+
+        public User GetUserByEmail(string email)
+        {
+            return dbContext.Users.Where(user => user.Email == email).FirstOrDefault();
```

Comment:
Here you can use the overloaded FirstOrDefault method without the Where method.

Reply:
fixed